### DISCUSSION TITLE
Enabling a second theme?

### DISCUSSION THREAD

**Discussion by jebofponderworthy** [searxng/searxng#3013] [Q&A] [answered]
I tried copying 'simple' to another folder, but did not find a way to enable the other folder as a second theme template.  Is there a simple way?

**Comment by return42** [comment, marked as answer]
Its not recommended to customize the theme by your own .. there is no official API, you need to maintain your own branch .. a lot of work and sooner or later you will get into trouble on updates.

In the long term, we would like to make our UI more modular to give maintainers more options ... but this is a longer process that will not be completed in the near future.
